.github/workflows/build-and-unit-tests.yaml

```diff
@@ -18,6 +18,7 @@ jobs:
           {arch: wormhole_b0, runs-on: ["wormhole_b0", "multi-chip-num-pcie-1", "multi-chip-num-chips-1"], name: N150},
           # N300
           {arch: wormhole_b0, runs-on: ["wormhole_b0", "multi-chip-num-pcie-1", "multi-chip-num-chips-2"], name: N300},
+          {arch: blackhole, runs-on: ["blackhole"], name: BH},
```

Comment:
can be removed, 
Will make relevant changes for CI in this PR
https://github.com/tenstorrent/tt-metal/pull/10888

Reply:
removed